tests/integrationv2/processes.py

```diff
@@ -20,7 +20,7 @@ class _processCommunicator(object):
     TLS clients (OpenSSL derivatives) to shut down before the handshake is complete.
 
     To prevent a premature shutdown, we need to wait until the handshake is complete
-    before writing to stdin. To accomplish this we `poll` stdout for a ready_to_send
+    before writing to stdin. To accomplish this we `poll` stdout for a send
```

Comment:
Is there a specific reason we are renaming `ready_to_send` to `send_marker_list` in these tests?

Reply:
I thought send_marker_list would be more readable because we are inputting a list of markers that indicate when we want to send data.